src/Resources/js/modules/combined-meal-service.ts

```diff
@@ -0,0 +1,109 @@
+export class CombinedMealService {
+    public static getCombinedMealDishes($meal: JQuery): Dish[] {
+        let dishes: Dish[] = [];
+        $meal.find('.meal-row').each(function () {
+            const $mealRow = $(this);
+            if (1 === $mealRow.data('combined')) {
```

Comment:
Shall we use "combined-meal" class rather than data attribute here?

Reply:
Yes, would like to do so, but after the merge in the feature branch would be better.